src/App.js

```diff
@@ -42,12 +42,17 @@ import { dataLoaded, isOnlineUpdate } from "./actions/index";
 import store from './store/index';
 import createRequestInstance from './api/axiosInstance';
 import hathorLib from '@hathor/wallet-lib';
-import { VERSION } from './constants';
+import { DEFAULT_SERVER, VERSION } from './constants';
 import LocalStorageStore  from './storage.js';
 
 
 hathorLib.storage.setStore(new LocalStorageStore());
 
+// set default server if needed
+if (!wallet.getServerURL()) {
+  hathorLib.wallet.changeServer(DEFAULT_SERVER);
```

Comment:
I would pass a list of hosts (instead of a single host).

Reply:
It only makes sense to do this if the wallet would automatically select the hosts, which we're not doing for now.